lib/bowlingGame.js

```diff
@@ -0,0 +1,115 @@
+class BowlingGame {
```

Comment:
Maybe naming of variables and functions could be a little clearer (or add an explanation in the readme). It's a little hard to understand the differences between 'strikeCounter' vs 'strikeCounter2' / 'strikeAndstrikeLogic2' vs strikeAndstrikeLogic3' etc.